Fortnox.NET.Tests/Fortnox.NET.Tests/PredefinedAccountsTest.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FortnoxNET.Communication;
+using FortnoxNET.Communication.PredefinedAccount;
+using FortnoxNET.Models.Account;
+using FortnoxNET.Models.PredefinedAccounts;
+using FortnoxNET.Services;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace FortnoxNET.Tests
+{
+    [TestClass]
+    public class PredefinedAccountsTest : TestBase
+    {
+        [TestMethod]
+        public async Task ItCanGetPredefinedAccounts()
+        {
+            var request = new PredefinedAccountListRequest(connectionSettings.AccessToken, connectionSettings.ClientSecret);
+            var response = await PredefinedAccountsService.GetPredefinedAccountsAsync(request);
+            
+            Assert.IsNotNull(response);
+            Assert.IsNotNull(response.Data);
+        }
+
+        [TestMethod]
+        public async Task ItCanGetAPredefinedAccount()
+        {
+            var accounts = await GetPredefinedAccounts();
+
+            if (!accounts.Any())
+            {
+                Assert.Inconclusive("No predefined accounts exist in the system");
+                return;
+            }
+            
+            var request = new PredefinedAccountListRequest(connectionSettings.AccessToken, connectionSettings.ClientSecret);
+            var response = await PredefinedAccountsService.GetPredefinedAccountAsync(
+                request,
+                accounts.First().Name    
+            );
+            
+            Assert.IsNotNull(response);
+            Assert.IsNotNull(response.Data);
+            Assert.AreEqual(accounts.First().Name, response.Data.Name);
+        }
+
+        [TestMethod]
+        public async Task ItCanUpdateAnExistingAccountTypeWithAPredefinedAccount()
+        {
+            var predefinedAccounts = await GetPredefinedAccounts();
+            var accounts = await GetAccounts();
+
+            if (!predefinedAccounts.Any())
+            {
+                Assert.Inconclusive("No predefined accounts exist in the system");
+                return;
```

Comment:
strictly speaking, there is no need for return statements, since the assert will break the controlflow

Reply:
below too

Reply:
resolved